Media/DisplayMedia/Strategies/AbstractStrategy.php

```diff
@@ -58,8 +58,14 @@ public function displayPreview(MediaInterface $media)
      */
     public function displayMediaForWysiwyg(MediaInterface $media, $format = '')
     {
-        return '<img class="tinymce-media" src="' . $this->getFileUrl($media->getThumbnail()) . '" alt="'
-            . $media->getAlt($this->request->getLocale()) . '" data-id="' . $media->getId() . '" />';
+        return $this->container->get('templating')->render(
```

Comment:
all right.

And what about only the first refactoring with the render method in the abstract class ?